tests/integration/path.rs

```diff
@@ -56,6 +59,22 @@ fn test_path_construction() {
     assert_eq!(PathBuf::empty(), []);
     assert_eq!(PathBuf::default(), []);
 
+    // Successful construction from std types.
+    #[cfg(feature = "std")]
```

Comment:
Shouldn't this also have cfg unix?

Reply:
Good catch, opened https://github.com/nicholasbishop/ext4-view-rs/pull/192 to check for this in the future.